pkg/registry/apis/folders/register.go

```diff
@@ -220,3 +218,38 @@ func authorizerFunc(ctx context.Context, attr authorizer.Attributes) (*authorize
 	}
 	return &authorizerParams{evaluator: eval, user: user}, nil
 }
+
+func (b *FolderAPIBuilder) Validate(ctx context.Context, a admission.Attributes, _ admission.ObjectInterfaces) error {
+	// name cannot be called "general"
+	id := a.GetName()
+	if id == invalidName {
+		return dashboards.ErrFolderInvalidUID
+	}
+
+	//max depth folder is 4
+	var maxFolderDepth = 4
```

Comment:
nit: Isn't this defined as a variable somewhere else? We could use it here as well to ensure we remain consistent.

Reply:
Seeing Bruno's comment above, same thing could be said for the constant for the General folder name.